### PR TITLE
Map 

### PR DESCRIPTION
**Pull request by arielrecto**
This Branch is based on Bike shop branch 

update:
add Map 
display all stores in the map
display temperature and user current address

### CODE REVIEW

**Review by mrabasto — DISMISSED** [review]

File: src/routes/home/map/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+			address.set(response.data.display_name)
+
+			console.log('====================================')
+			console.log(response.data)
+			console.log('====================================')
+		} catch (error) {
+			console.log(error)
+		}
+	}
+
```
Comment (mrabasto) [inline src/routes/home/map/+page.svelte]: Remove console.log, then use errorToast for displaying errors

**Review by mrabasto — CHANGES_REQUESTED** [review]

File: src/routes/home/map/+page.svelte (hunk 1 above)
Comment (arielrecto) [reply src/routes/home/map/+page.svelte]: I commit the changes

**Review by mrabasto — DISMISSED** [review]

**Comment by mrabasto** [comment]
LGTM

**Review by mrabasto — APPROVED** [review]

**Review by mrabasto — DISMISSED** [review]
LGTM

**Pull request merged**